doc/stdlib/index-list.html.template

```diff
@@ -595,6 +595,7 @@ through the <tt>Require Import</tt> command.</p>
     theories/Reals/SeqSeries.v
     theories/Reals/Sqrt_reg.v
     theories/Reals/Rlogic.v
+    theories/Reals/Rregisternames.v
```

Comment:
If you had put this file in the hidden index, it wouldn't be listed in the doc.

Reply:
Actually I want it to be listed in the doc so that it is clear that this is part of the library interface and that if people use symbols of teh reals library, in plugins they should go in here. The issue I have is that the doc is empty - it doesn't show the name registrations, just a headline.

Reply:
@MSoegtropIMC coqdoc only knows about a subset of commands. This one was considered an internal command (in fact it is only useful to plugin authors), so it was never added to coqdoc. You could open an issue to have this discussed.

Reply:
I will do so. I consider the registered names as part of the interface of the library, because plugin authors interface to it. Knowing the names of 0 and 1 might be useful for more than one plugin. Should I meanwhile add some doc visible comment stating what is in the file?

Reply:
Sure!